### PR TITLE
Disable GSUB5 optimization

### PR DESCRIPTION
**Pull request by simoncozens**
Disables optimisation as mentioned in #2539.

### CODE REVIEW

**Comment by behdad** [comment]
Ideally just do it for GPOS7.

**Comment by behdad** [comment]
Lgtm.

File: Tests/feaLib/builder_test.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ class BuilderTest(unittest.TestCase):
         LigatureSubtable AlternateSubtable MultipleSubstSubtable 
         SingleSubstSubtable aalt_chain_contextual_subst AlternateChained 
         MultipleLookupsPerGlyph MultipleLookupsPerGlyph2 GSUB_6_formats
-        GSUB_5_formats delete_glyph STAT_test STAT_test_elidedFallbackNameID
+        delete_glyph STAT_test STAT_test_elidedFallbackNameID
```
Comment (khaledhosny) [inline Tests/feaLib/builder_test.py:81]: Can we get this test back?

**Comment by anthrotype** [comment]
LGTM
@simoncozens ready for merge? should you maybe rename the PR title to clarify this only affects GPOS7 and not also GSUB5?

**Comment by anthrotype** [comment]
thanks, going to cut a new ft release now

**Comment by simoncozens** [comment]
Oh bother. You're right.

**Pull request merged**